web/sites/default/settings.php

```diff
@@ -42,17 +42,28 @@
   'bower_components',
 ];
 
+// $settings['updates_log_site'] = 'my-project-name';
+// In Silta it is set by default as github repo name.
+
+// $settings['updates_log_env'] = getenv('ENVIRONMENT_NAME');
+// In Silta it is set by default as ENVIRONMENT_NAME
+
+// $settings['updates_log_disabled'] = TRUE;
```

Comment:
Let's not bring that double negation variable into the template! Instead, let's implement `config_split` properly either via `config` or `recipes`.